src/models/users/useUserMetadataSchemas.js

```diff
@@ -46,16 +46,18 @@ export default function useUserMetadataSchemas(displayedUserId) {
             currentPlatformEnablingField,
             'value',
           ]);
-          return isEnabled
-            ? createFieldSchema(fieldTypes.string, {
-                name: intelligentAgent?.userMetadataKey,
-                labelId: intelligentAgent?.viewLabelId,
-                editLabelId: intelligentAgent?.editLabelId,
-                icon: intelligentAgent?.icon,
-              })
-            : null;
+          return isEnabled;
         },
-        [],
+      );
+      const intelligentAgentFields = map(
+        enabledIntelligentAgentFields,
+        intelligentAgent =>
+          createFieldSchema(fieldTypes.string, {
+            name: intelligentAgent?.userMetadataKey,
+            labelId: intelligentAgent?.viewLabelId,
+            editLabelId: intelligentAgent?.editLabelId,
+            icon: intelligentAgent?.icon,
```

Comment:
Is the optional chaining operator necessary?

Reply:
I suppose not!